src/main.rs

```diff
@@ -25,7 +28,10 @@ struct State<'a, 'b> {
 
 impl<'a, 'b> ggez::event::EventHandler for State<'a, 'b> {
     fn mouse_button_down_event(&mut self, _ctx: &mut Context, button: MouseButton, x: f32, y: f32) {
-        if button == MouseButton::Left && *self.world.read_resource::<PlayState>() == PlayState::Play {
+        // TODO: Move this to a system.
+        // If the player clicks on an open spot on the grid and has enough bits, then build a tower.
+        if button == MouseButton::Left && *self.world.read_resource::<PlayState>() == PlayState::Play &&
```

Comment:
Can we put each `&&` on a separate line?

Reply:
Done!